home/src/main/resources/config/example.runtime.properties

```diff
@@ -153,3 +153,8 @@ proxy.eligibleTypeList = http://www.w3.org/2002/07/owl#Thing
 # This should not be used with languages.forceLocale, which will override it.
 #
 # languages.selectableLocales = en, es, fr
+
+# Triple pattern fragments is a very fast, very simple means for querying a triple store.
+# The triple pattern fragments API in VIVO puts little load on the server, providing a simple means for getting data from the triple store. The API has a web interface for manual use, can be used from the command line via curl, and can be used by programs.
+
+# tpf.activeFlag = true
```

Comment:
I think we want the default to be disabled, due to the concern of inadvertently exposing otherwise protected content through the TPF.

Reply:
it is commented out so the way the code works if it doesn't exist or is anything but true then it is disabled.  have it the way it is to make it easy to enable if wanted.